internal/bsi/bsi.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package bsi.v1;
```

Comment:
Considering that we will probably need to interact with other services like GAR, I wonder if it makes sense to split these proto pkgs out by the cloud service they are managing or just make a generic one like: `sci`: `Substratus Cloud Interface`. I wouldnt want to go all micro-servicey here... I think that its OK to keep this as `bsi` and just have a single server process implement multiple gRPC service interfaces... but have a slight preference for the combined (`sci`) gRPC service interface.

Reply:
I started that path initially but it felt like the start of boiling the ocean to build an interface whose scope is all APIs of all providers. I think confining it to substratus cloud interface is a good way to keep the scope well understood and under control.